<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleEditor</class>
 <widget class="QWidget" name="StyleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupStyles">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Styles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QListWidget" name="stylesNames">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupFont">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Apparence</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QFontComboBox" name="fontName"/>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnBold">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Bold</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/icons/bold.svg</normaloff>:/icons/bold.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="fontSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnColor">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="btnItalic">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Italic</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/icons/italic.svg</normaloff>:/icons/italic.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QPushButton" name="btnAdvanced">
        <property name="toolTip">
         <string>Alignment, margins, ...</string>
        </property>
        <property name="text">
         <string>Advanced...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stylesNames</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>StyleEditor</receiver>
   <slot>styleSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontName</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>StyleEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>StyleEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColor</sender>
   <signal>released()</signal>
   <receiver>StyleEditor</receiver>
   <slot>chooseColour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdvanced</sender>
   <signal>released()</signal>
   <receiver>StyleEditor</receiver>
   <slot>advancedConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBold</sender>
   <signal>toggled(bool)</signal>
   <receiver>StyleEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnItalic</sender>
   <signal>toggled(bool)</signal>
   <receiver>StyleEditor</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply()</slot>
  <slot>chooseColour()</slot>
  <slot>advancedConfig()</slot>
  <slot>styleSelected()</slot>
 </slots>
</ui>
